### PR TITLE
fix(Rich Text Editor): markdown de/serialize for list

### PR DESCRIPTION
**Pull request by bojangles-m**
Refactored to work with plates listItemChild which is now considered in deserialization.

### CODE REVIEW

File: src/components/RichTextEditor/serializer/markdown/__tests__/fixtures/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        children,
+    };
+};
+
+export const createOL = <T>(children: Array<T>) => createListElement('ol', children);
+export const createUL = <T>(children: Array<T>) => createListElement('ul', children);
+
+export const createLI = <T>(children: Array<T>) => createListElement('li', children);
+
+export const createLIC = (text: string) => createListElement('lic', [{ text }]);
```
Comment (SamuelAlev) [inline src/components/RichTextEditor/serializer/markdown/__tests__/fixtures/helpers.ts:15]:
```suggestion
export const createOl = <T>(children: Array<T>) => createListElement('ol', children);
export const createUl = <T>(children: Array<T>) => createListElement('ul', children);

export const createLi = <T>(children: Array<T>) => createListElement('li', children);

export const createLic = (text: string) => createListElement('lic', [{ text }]);
```

File: src/components/RichTextEditor/serializer/markdown/__tests__/fixtures/unorderedList.ts [hunk 2]
```diff
@@ -12,86 +14,97 @@ export const unorderedListMarkdown = `
 
 
 `;
+const unorderedListTree_0 = [
```
Comment (SamuelAlev) [inline src/components/RichTextEditor/serializer/markdown/__tests__/fixtures/unorderedList.ts:17]:
```suggestion
const unorderedListTree0 = [
```
Why the underscore? (same for the others)

File: src/components/RichTextEditor/serializer/markdown/remarkFondue/index.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    extensions.set('listExtension', remarkList());
+
+    for (const extension of extensions.values()) {
+        extension(tree);
+    }
+};
+
+export default function remarkFondue() {
+    return transformer;
+}
```
Comment (SamuelAlev) [inline src/components/RichTextEditor/serializer/markdown/remarkFondue/index.ts:20]:
```suggestion
export const remarkFondue = () => transformer;
```

Comment (bojangles-m) [reply src/components/RichTextEditor/serializer/markdown/remarkFondue/index.ts]: Updated

File: src/components/RichTextEditor/serializer/markdown/__tests__/fixtures/unorderedList.ts (hunk 2 above)
Comment (bojangles-m) [reply src/components/RichTextEditor/serializer/markdown/__tests__/fixtures/unorderedList.ts]: Renamed

File: src/components/RichTextEditor/serializer/markdown/__tests__/fixtures/helpers.ts (hunk 1 above)
Comment (bojangles-m) [reply src/components/RichTextEditor/serializer/markdown/__tests__/fixtures/helpers.ts]: Renamed

**Review by SamuelAlev — APPROVED** [review]

**Pull request merged**